filtering.py

```diff
@@ -167,7 +214,7 @@ def find_cmds(message, user, roomid):
     origin_room = None
     match = re.findall(r"\(|\)", command_string)
 
-    if perms == 'dev' and roomid == 'jN7Ht3giH9EDBvpvnqRB' and match != []:
+    if perms == 'dev' and roomid == 'jN7Ht3giH9EDBvpvnqRB12' and match != []:
```

Comment:
should change this on the README as well

Reply:
Its a typo